modin/pandas/indexing.py

```diff
@@ -200,19 +206,24 @@ def _generate_view(self, row_lookup, col_lookup):
         for i in col_lookup["partition"]:
             col_lengths[i] += 1
 
+        row_lengths_oid = ray.put(np.array(row_lengths))
+        col_lengths_oid = ray.put(np.array(col_lengths))
+
         row_metadata_view = _IndexMetadata(
-            coord_df_oid=row_lookup, lengths_oid=row_lengths)
+            coord_df_oid=row_lookup, lengths_oid=row_lengths_oid)
```

Comment:
Can we modify the metadata class to take in an `np.array` here, as well as an OID, instead of using `ray.put`?

Reply:
I think it should be just oid here because the keyword arg is `_oid`. When we refactor `IndexMetaData` (which will be another PR and coming soon), we can make it taking `np.array`. I don't want this to be a refactoring PR.

Reply:
Can we file an issue or mark a todo in the code in that case?

Reply:
Yep. I think @devin-petersohn will soon write up a design doc for the new index (using two B+Tree) that will basically re-write `_IndexMetaData`.